routes/router.js

```diff
@@ -637,13 +633,16 @@ router.route('/uploadFile')
                         // need the index, so use a regular for loop
                         for (var i = 0; i < vals.length; i++) {
 
-                            // get the p element
-                            var val = vals[i].firstChild; 
-                            
-                            // if the cell is empty then continue
-                            if (!val) {
-                                continue;
+                            var text_elements = vals[i].querySelectorAll('p');
+
+                            // extract and append all the text from all the text elements
+
+                            var text = '';
+                            for (let val of text_elements) {
+                                text = text.concat(val.text);
```

Comment:
Is it possible the content would be an empty string/whitespace? If so, it might be worth handling that case differently.